examples/plot_mne_sample_gridsearch.py

```diff
@@ -0,0 +1,99 @@
+"""
+====================================================================
+Optimized feature extraction from MEG time series for classification
```

Comment:
I would say:

"Use scikit-learn GridSearchCV with FeatureExtractor for setting parameters"

this is clear.